contracts/main/bridge/FathomBridge.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import "./AsterizmClientUpgradeableTransparency.sol";
+import "../interfaces/IStablecoinAdapter.sol";
+import "../interfaces/IBookKeeper.sol";
+import "../interfaces/ICagable.sol";
+import "../interfaces/IFathomBridge.sol";
+import "../utils/SafeToken.sol";
+
+/**
+ * @title FathomBridge
+ * @notice A contract which acts as an entrypoint for bridging Fathom Stablecoin.
+ * It has the ability to mint FXD in the source chain and burn FXD in the destination chain.
+ * _initializerLib contract needs to exist prior to the deployment of this smart contract. 
+ * Please refer to below documentation for the list of deployed AsterizmInitializerLib contract
+ * https://docs.asterizm.io/technical-reference/mainnet
+ * This contract, like FathomProxyAdmin&FathomProxyFactory, has its own owner outside of Fathom Protocol's accessControlSystem due to inheritance of AsterizmClientUpgradeableTransparency
+ * Have the off-chain client module as the OZ owner of this contract to manage bridge related operations
+ */
+
+contract FathomBridge is AsterizmClientUpgradeableTransparency, PausableUpgradeable, IFathomBridge, ICagable {
+    using SafeToken for address;
+    IBookKeeper public bookKeeper;
+    IStablecoinAdapter public stablecoinAdapter;
+    uint256 public fixedBridgeFee; // fixed fee [wad]
+    uint256 public live; // Active Flag
+    bool public isDecentralizedMode;
+    mapping(address => bool) public whitelisted;
+
+    event LogAddToWhitelist(address indexed _user);
```

Comment:
Let's move events to the `IFathomBridge` interface. It will help with readability

Reply:
Okay, makes sense.